src/components/header/Location.css

```diff
@@ -0,0 +1,17 @@
+.location{
+    margin-left: 10px;
+    cursor: pointer;
```

Comment:
I saw at some places you have used **_cursor:pointer_** property. May be you can have a small class for it